packages/office-ui-fabric-react/src/components/Button/Button.test.tsx

```diff
@@ -68,4 +68,31 @@ describe('DefaultButton', () => {
       expect(button.getAttribute('aria-haspopup')).to.equal(null);
     });
   });
+
+  it('Providing onClick and menuProps renders a SplitButton', () => {
+    const button = ReactTestUtils.renderIntoDocument<any>(
+      <DefaultButton
+        data-automation-id='test'
+        text='Create account'
+        onClick={ () => alert('Clicked') }
+        menuProps={ {
+          items: [
+            {
+              key: 'emailMessage',
+              name: 'Email message',
+              icon: 'Mail'
+            },
+            {
+              key: 'calendarEvent',
+              name: 'Calendar event',
+              icon: 'Calendar'
+            }
+          ]
+        } }
+      />
+    );
+    const renderedDOM = ReactDOM.findDOMNode(button as React.ReactInstance);
+    expect(renderedDOM.tagName).equal('DIV', 'A Button with a menuProps and an onClick renders as a SplitButton');
```

Comment:
The SplitButton is the only button that renders two buttons wrapped inside a `div`.